sessionctx/variable/variable.go

```diff
@@ -614,6 +618,22 @@ func GetSysVars() map[string]*SysVar {
 	return m
 }
 
+// OrderByDependency orders the vars by dependency. The depended sys vars are in the front.
+// Unknown sys vars are treated as not depended.
+func OrderByDependency(names map[string]string) []string {
```

Comment:
What if there are dependencies between the `Depended` variables?

Reply:
There are not for now. If that does in the future, I'll make it supported. It will make things more complicated and I don't want to make it complicated too early.

Reply:
It makes sense, thanks.